internal/pkg/pluginengine/pluginengine_test.go

```diff
@@ -28,7 +28,7 @@ var _ = Describe("Pluginengine", func() {
 	BeforeEach(func() {
 		defer GinkgoRecover()
 
-		smgr, err = statemanager.NewManager()
+		smgr, err = statemanager.NewManager("local")
```

Comment:
here should use const `Local` instead of "local"

Reply:
This is handled inside `backend.GetBackend()`. The user of `statemanager` should not care too much about the implementation of `backend`.